### ISSUE TITLE
Nextcloud container doesn't start because of database not ready condition

### ISSUE THREAD

**Issue by InterFelix** [nextcloud/helm#150]
I have tried to deploy Nextcloud on the latest chart version (2.8.3) using both MariaDB and PostgreSQL, and all attempts have failed so far because of the following condition:
All resources are created correctly, the database (MariaDB or Postgres) comes up and is ready to accept connections. I have tested and in both cases I could actually connect to the databases.
However, the Nextcloud container itself never comes up, because the condition "postgresql-isready" or "mariadb-isready" (from the initContainer) is never fulfilled. 
This has to be some kind of bug with the latest few versions of the chart since I have successfully deployed nextcloud from the chart before.

**Comment by DebakelOrakel** [comment]
Have the same problem, also with mariadb, but only after i restart the nextxloud pod.
On the first deployment everything works fine.

**Comment by jessebot** [comment]
Closing as stale due to no response in over a year. If anyone else is still having this issue, please open a fresh Issue using our new Issue template and please include your values.yaml (after removing any sensitive data), so that we can better reproduce the issue. Happy to keep helping if anyone needs it though! Kind regards 🙏
